install/install.py

```diff
@@ -179,6 +179,28 @@ def install(self, cmd_arguments):
         # db_run script
         self.run_command("sudo sh %s" % (os.path.join(self.scripts_path, "db_run.sh")))
 
+
+class Colorizer:
+    RED = '\033[91m'
+    GREEN = '\033[92m'
+    YELLOW = '\033[93m'
+    BLUE = '\033[34m'
+    PURPLE = '\033[95m'
+    CYAN = '\033[36m'
+    BOLD = '\033[1m'
+    END = '\033[0m'
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def colorize(cls, color, string, bold=False):
+        if bold:
+            print(cls.BOLD + eval("cls." + color) + string + cls.END + cls.END)
+        else:
+            print(eval("cls." + color) + string + cls.END)
```

Comment:
This commit has been squashed and modified with next commit and no longer exists.